Sources/Hyperwallet.swift

```diff
@@ -951,6 +951,30 @@ public final class Hyperwallet: NSObject {
                                        completionHandler: transferMethodConfigurationKeyResponseHandler(completion))
     }
 
+    /// Returns the transfer method configuration processing times, and fees for the User that is associated
+    /// with the authentication token returned from
+    /// `HyperwalletAuthenticationTokenProvider.retrieveAuthenticationToken(_ : @escaping CompletionHandler)`.
+    ///
+    /// The `completion: @escaping (HyperwalletTransferMethodConfigurationKey?, HyperwalletErrorType?) -> Void`
+    /// that is passed in to this method invocation will receive the successful
+    /// response(HyperwalletTransferMethodConfigurationFeeAndProcessingTime) or
+    /// error(HyperwalletErrorType) from processing the
+    /// request.
+    ///
+    /// This function will request a new authentication token via `HyperwalletAuthenticationTokenProvider`
+    /// if the current one is expired or is about to expire.
+    ///
+    /// - Parameters:
+    ///   - request: containing the transfer method configuration key query
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+    public func retrieveFeeAndProcessingTime(
```

Comment:
When I suggested this name, I also suggested a different query that will just fetch fee and processing time but since your query is fetching transferMethodType Fee and Processing time kindly update the name appropriately

Reply:
Renamed to `retrieveTransferMethodTypesFeesAndProcessingTimes`